application/src/main/java/org/togetherjava/tjbot/commands/help/BookmarksHelper.java

```diff
@@ -0,0 +1,109 @@
+package org.togetherjava.tjbot.commands.help;
+
+import org.togetherjava.tjbot.db.Database;
+import org.togetherjava.tjbot.db.generated.tables.records.BookmarksRecord;
+import org.togetherjava.tjbot.db.generated.tables.records.HelpThreadsRecord;
+
+import javax.annotation.Nullable;
+
+import java.time.Instant;
+import java.time.temporal.ChronoUnit;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+import static org.togetherjava.tjbot.db.generated.Tables.BOOKMARKS;
+import static org.togetherjava.tjbot.db.generated.Tables.HELP_THREADS;
+
+/**
+ * This is a utility class for interacting with the database
+ */
+public class BookmarksHelper {
+
+    private BookmarksHelper() {
+        throw new IllegalStateException("Utility class");
+    }
+
+    protected static Optional<HelpThreadsRecord> getHelpThread(Database database, long channelId) {
+        return database.read(context -> context.selectFrom(HELP_THREADS)
+            .where(HELP_THREADS.CHANNEL_ID.eq(channelId))
+            .fetch()
+            .stream()
+            .findFirst());
+    }
+
+    protected static Optional<BookmarksRecord> getBookmark(Database database, long creatorId,
+            long channelId) {
+        return database.read(context -> context.selectFrom(BOOKMARKS)
+            .where(BOOKMARKS.CREATOR_ID.eq(creatorId), BOOKMARKS.CHANNEL_ID.eq(channelId))
```

Comment:
```suggestion
            .where(BOOKMARKS.CHANNEL_ID.eq(channelId))
```
Channel ID's are unique, shouldn't need the creator ID and the channel ID

Reply:
Nope that is correct. You need the creator ID because multiple people can bookmark the same channel but with different notes/expire dates

Reply:
Ah my brain, I didn't read it correctly. Thought it was the channel creator's ID. 
Can you name it "user_id" or something? Might be more appropriate I'd say

Reply:
What does it create then? A "bookmark" yes, but it's more of a link to a help channel

I'd like a second opinion on this matter xD

Reply:
i had the same confusion when reviewing the database model (see my comments on the sqf file). its confusing whether it talks about the channel, the bookmark or possibly even sth else. same with the other entries in the database